src/sage/combinat/matrices/hadamard_matrix.py

```diff
@@ -3055,8 +3059,8 @@ def skew_hadamard_matrix(n, existence=False, skew_normalize=True, check=True):
     Tries to construct a skew Hadamard matrix.
 
     A Hadamard matrix `H` is called skew if `H=S-I`, for `I` the identity matrix
-    and `-S=S^\top`. Currently all orders `\le 1000` for which a construction is
-    known are implemented. For `n > 1000`, only some orders are available.
+    and `-S=S^\top`. Currently all orders `\le 1200` for which a construction is
+    known are implemented. For `n > 1200`, only some orders are available.
```

Comment:
this is correct, but the title of the PR indicates 1201, so may be you should use 1201 here as well ?

Reply:
$<1201$ is the same as $\leq 1200$ 😄 
Sorry.